src/test/java/com/wikia/webdriver/testcases/discussions/CategoriesTests.java

```diff
@@ -88,8 +88,7 @@ private void deleteCategoriesMobile() {
   }
 
   private void deleteCategories(String wikiName) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "deleteCategories" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)